app/assets/javascripts/links.js

```diff
@@ -0,0 +1,12 @@
+$(document).ready(function() {
```

Comment:
This will enable infinite scrolling for all pages where there is pagination. It should be on links page only as of now. 

To fix this give an id to div that contains all links e.g `bookmarks` and you can check like
`($('.pagination').length >0 && $('#bookmarks').length > 0` in the below condition